# RNA-Seq and metabolic flux analysis of Tetraselmis sp. M8 during nitrogen starvation reveals a two-stage lipid accumulation mechanism - Supplementary Data

This GitHub repository provides the supplementary data referenced in the publication cited below.

## How to use and cite these files 

All files are publicly available and can be used for further research or other applications. However, if you utilize these resources in your work, we kindly request that you cite our original publication.

**RNA-Seq and metabolic flux analysis of Tetraselmis sp. M8 during nitrogen starvation reveals a two-stage lipid accumulation mechanism.** David K.Y. Lim, Holger Schuhmann, Skye R. Thomas-Hall, Kenneth C.K. Chan, Taylor J. Wass, Felipe Aguilera, T. Catalina Adarme-Vega, Cristiana G.O. Dal'Molin, Glen J. Thorpe, Jacqueline Batley, David Edwards, Peer M. Schenk. *Bioresource Technology* **244**:1281-1293 (2017). https://doi.org/10.1016/j.biortech.2017.06.003.

**Abstract**

To map out key lipid-related pathways that lead to rapid triacylglyceride accumulation in oleaginous microalgae, RNA-Seq was performed with Tetraselmis sp. M8 at 24 h after exhaustion of exogenous nitrogen to reveal molecular changes during early stationary phase. Further gene expression profiling by quantitative real-time PCR at 16–72 h revealed a distinct shift in expression of the fatty acid/triacylglyceride biosynthesis and β-oxidation pathways, when cells transitioned from log-phase into early-stationary and stationary phase. Metabolic reconstruction modeling combined with real-time PCR and RNA-Seq gene expression data indicates that the increased lipid accumulation is a result of a decrease in lipid catabolism during the early-stationary phase combined with increased metabolic fluxes in lipid biosynthesis during the stationary phase. During these two stages, Tetraselmis shifts from reduced lipid consumption to active lipid production. This process appears to be independent from DGAT expression, a key gene for lipid accumulation in microalgae.

## Author contact

- David K.Y. Lim (first author - PhD student)
- [Felipe Aguilera](mailto:f.aguilera@uq.edu.au) (PhD student)
- [Peer M. Schenk](mailto:p.schenk@uq.edu.au) (corresponding author)
- Lab website: https://www.aguileralab.com

## How to download data files

The files in this repository are ready for use. Simply click on the file, and you will be directed to the GitHub webpage to save it to your device.

## Index of data file contents

[01-Supplementary-Figures.pdf](https://github.com/faguil/Tetraselmis-M8-RNA-Seq/blob/main/01-Supplementary-Figures.pdf) contains supplementary figures used in this study.

- **Figure S1.** Computational predictions of cytosolic, membrane-bound and secreted proteins for each species.
- **Figure S2.** GO annotations of genes that arose along the stems leading to bilaterian (PS7, red) and molluscan (PS10, blue) last common ancestors.
- **Figure S3.** Phylogenetic relationships among bivalve and gastropod lineages based on 122 gene families that encode secreted proteins.
- **Figure S4.** GO term (biological process and molecular function) enrichment in newly gained secreted gene families across conchiferan evolution.
- **Figure S5.** Expression profiles of genes encoding secreted proteins – the mantle secretome – of eight bivalve and two gastropod species.
- **Figure S6.** Expression levels of co-opted, lineage-specific and species-specific genes encoding secreted proteins.
- **Figure S7.** Phylogenetic analysis of carbonic anhydrase domain-containing proteins.
- **Figure S8.** Phylogenetic analyses of SPARC domain-containing proteins and zona pellucida-like domain-containing proteins.
- **Figure S9.** Phylogenetic analysis of tyrosinase domain-containing proteins.
- **Figure S10.** Phylogenetic analysis of glycoside hydrolase family 18 domain-containing proteins.
- **Figure S11.** Phylogenetic analyses of polysaccharide deacetylase domain-containing proteins and chitin-binding domain-containing proteins.
- **Figure S12.** Changes in the prediction of gene families using different MCL inflation values.
